src/chartiq/components/chartTitle/chartTitle.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect, useRef } from 'react';
+import "./chartTitle.scss";
+import { ChartContext } from '../../contexts/index';
+
+export default function ChartTitle() {
+	const { theme, stx } = React.useContext(ChartContext);
+  const [upFlag, setUpFlag] = useState(false);
+  const [data, setData] = useState({
+		difference: 0,
+		close: 0,
+		changedValuePct: 0,
+		symbol: '',
+	});
+
+	function usePrevious(value) {
+		const ref = useRef();
+		useEffect(() => {
+			ref.current = value;
+		});
+		return ref.current;
+	}
+
+	const prevAmount = usePrevious(data);
+
+	function closureDataOrserver() {
+		stx.append("createDataSet", () => {
+			const currentQuote = stx.getFirstLastDataRecord(
+				stx.chart.dataSet,
+				"Close",
+				true
+			);
+
+			setData((prev) => {
+				return {
+					...prev,
+					symbol: stx.chart.symbol,
+					close: currentQuote.Close,
+					difference: currentQuote.Close - prev.close,
+					changedValuePct: ((currentQuote.Close - prev.close) / currentQuote.Close) * 100,
```

Comment:
What is the reason for not having the `upFlag` simply set here and going through trouble of saving data also in ref?
```
upFlag: currentQuote.Close > prev.close
```